src/utils/validation.ts

```diff
@@ -314,9 +316,76 @@ export async function getDefaultConfig() {
 }
 
 async function getValidatorInstance() {
-  const {MonokleValidator, ResourceParser, SchemaLoader, RemotePluginLoader, requireFromStringCustomPluginLoader, DisabledFixer, AnnotationSuppressor, FingerprintSuppressor, processRefs} = await import('@monokle/validation');
+  const {MonokleValidator, ResourceParser, SchemaLoader, RemotePluginLoader, requireFromStringCustomPluginLoader,  DisabledFixer, AnnotationSuppressor, FingerprintSuppressor, processRefs} = await import('@monokle/validation');
   const {fetchOriginConfig} = await import('@monokle/synchronizer');
 
+
+const fixer = {
```

Comment:
Maybe this can be extracted as separate function to not decrease `getValidatorInstance()` readability? Also indentation seems to be off 🤔